pkg/resources/user_public_keys.go

```diff
@@ -60,10 +62,12 @@ func UserPublicKeys() *schema.Resource {
 }
 
 func checkUserExists(db *sql.DB, name string) (bool, error) {
```

Comment:
this seems like a rather pointless function, we should probably remove this entirely. But not important right now

Reply:
I am not sure, but I tried not to modify the existing logic in this PR.